primitives/version/versions.go

```diff
@@ -20,23 +20,64 @@
 
 package version
 
+import (
+	"github.com/berachain/beacon-kit/primitives/bytes"
+	"github.com/berachain/beacon-kit/primitives/common"
+)
+
 const (
-	// Phase0 is the first version of the Beacon Chain.
-	Phase0 uint32 = 0
-	// Altair is the first hardfork of the Beacon Chain.
-	Altair uint32 = 1
-	// Bellatrix is the second hardfork of the Beacon Chain.
-	Bellatrix uint32 = 2
-	// Capella is the third hardfork of the Beacon Chain.
-	Capella uint32 = 3
-	// Deneb is the first version of Deneb, used for genesis of Berachain mainnet.
-	Deneb uint32 = 4
-	// Deneb1 is the first hardfork of Deneb on Berachain mainnet (TBD if used).
-	// There may also be Deneb2, Deneb3, etc. hardforks.
-	Deneb1 uint32 = 260
-	// Electra is the first version of Electra on Berachain mainnet.
-	Electra uint32 = 5
-	// Electra1 is the first hardfork of Electra on Berachain mainnet (TBD if used).
-	// There may also be Electra2, Electra3, etc. hardforks.
-	Electra1 uint32 = 261
```

Comment:
the reason we dropped `Electra1` is that we don't need to show anymore how to get the next?

Reply:
Don't mind keeping this in or out tbh

Reply:
Kept in